usr/lib/common/mech_aes.c

```diff
@@ -3602,27 +3606,31 @@ CK_RV aes_gcm_decrypt_update(STDLL_TokData_t *tokdata, SESSION *sess,
      * that means it's tag data, not encrypted plaintext.
      * Hence we'll keep at least tag data size in the context buffer */
 
+    aesgcm = (CK_GCM_PARAMS *) ctx->mech.pParameter;
     context = (AES_GCM_CONTEXT *) ctx->context;
-    total = (context->len + in_data_len);
 
-    aesgcm = (CK_GCM_PARAMS *) ctx->mech.pParameter;
+    total = (context->len + in_data_len);
```

Comment:
brackets around the addition are not needed.

Reply:
OK, removed.